pytket/docs/backends.rst

```diff
@@ -3,11 +3,11 @@ pytket.backends
 
 Contains :py:class:`Backend` abstract class and associated methods. In pytket a :py:class:`Backend` represents an interface between pytket and a quantum device or simulator. Different backends are defined in the various pytket extension modules and inherit from the core pytket :py:class:`Backend` class.
 
-There are several `example notebooks <https://tket.quantinuum.com/examples>`_ on pytket :py:class:`Backend`\s. If you are interested in developing your own :py:class:`Backend` or pytket extension then see the `creating backends <https://tket.quantinuum.com/examples/creating_backends.html>`_ tutorial.
+There are several `example notebooks <https://tket.quantinuum.com/examples>`_ on pytket :py:class:`Backend`\s. If you are interested in developing your own :py:class:`Backend` or pytket extension then see the `creating backends <https://tket.quantinuum.com/user-guide/examples/creating_backends.html>`_ tutorial.
```

Comment:
I get 404 with this URL.

Reply:
Thanks for spotting... fixed in [e197d77](https://github.com/CQCL/tket/pull/1571/commits/e197d7722e224291278006484fe5f9a84cf1f7c6)